### DISCUSSION TITLE
DynamicServerError

### DISCUSSION THREAD

**Discussion by chris-nowicki** [vercel/community#2422] [Bug Report]
### Problem Description

I am currently using the beta cron jobs feature with NextJS 13.3.1.  Everything has been working fine, no changes made except for some files that are not related.  I now get this error on build.  Note the scripts in /api/cron/** work just fine when I run the dev environment locally:

![Screenshot 2023-04-25 at 11 08 34 AM](https://user-images.githubusercontent.com/102450568/234321206-8115e0d4-1625-4deb-b6d4-7ff5818fb39a.png)

Not sure what is going on? Maybe it is because it is still in beta? I even reverted back to the commit before it started happening and no luck.  I have a support ticket in but thought I'd ask here as well!

**Comment by amyegan** [comment]
Thanks for working with the Customer Success team to find a solution. I'm glad to see that updating to `13.4` worked 🎉
